container-chains/runtime-templates/simple/src/weights/pallet_balances.rs

```diff
@@ -156,4 +156,20 @@ impl<T: frame_system::Config> pallet_balances::WeightInfo for SubstrateWeight<T>
 		Weight::from_parts(7_531_000, 1501)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 	}
-}
\ No newline at end of file
+    fn burn_allow_death() -> Weight {
```

Comment:
we copied this from somewhere¿?

Reply:
Yes, I believe from westend or rococo, I didn't run the benchmarks and I needed this for the code to compile.